addme

```diff
@@ -0,0 +1,105 @@
+#/bin/bash
+
+#set -x 
+
+OS=`uname -s`
+mygid=7903
+myuid="556"
+myuname="jeffcof"
```

Comment:
For the most part, we don't really care what our uid and gid are.  Just let system pick one is ok.

Reply:
yes, you can use uid uname options to define what you want.